src/components/Startpage/Startpage.tsx

```diff
@@ -0,0 +1,255 @@
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    let endedAuctionsElement = (
+        <div className="cards-wrapper">{
+            endedAuctions.map(getAuctionElement)}
+        </div>
+    )
+
+    let newPlayersElement = (
+        <div className="cards-wrapper">{
+            newPlayers.map(newPlayer =>
+            (
+                <div className="card-wrapper" key={newPlayer.uuid}>
+                    <Link className="disable-link-style" to={`/player/${newPlayer.uuid}`}>
+                        <Card className="card">
+                            <Card.Header style={{ padding: "10px" }}>
+                                <div style={{ float: "left" }}>
+                                    <img crossOrigin="anonymous" src={newPlayer.iconUrl} width="32" height="32" alt="" style={{ marginRight: "5px" }} loading="lazy" />
+                                </div>
+                                {newPlayer.name}
+                            </Card.Header>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    let newItemsElement = (
+        <div className="cards-wrapper">{
+            newItems.map(newItem =>
+            (
+                <div className="card-wrapper" key={newItem.tag}>
+                    <Link className="disable-link-style" to={`/item/${newItem.tag}`}>
+                        <Card className="card">
+                            <Card.Header style={{ height: "100%" }}>
+                                <div style={{ float: "left" }}>
+                                    <img crossOrigin="anonymous" src={newItem.iconUrl} width="32" height="32" alt="" style={{ marginRight: "5px" }} loading="lazy" />
+                                    {newItem.name}
+                                </div>
+                            </Card.Header>
+                        </Card>
+                    </Link>
+                </div>
+            ))}
+        </div>
+    )
+
+    //style={{ display: "flex", flexWrap: "wrap", justifyContent: "space-between", alignItems: "stretch" }}
+
+    return (
+        <div className="startpage">
+
+            <div className="startpage-list-element-wrapper">
+                <Card style={{ width: "100%" }}>
+                    <Card.Header>
+                        <Card.Title><AnnouncementIcon/><span style={{ color: "#40ff00" }}> News / Announcements</span></Card.Title>
+                        <Card.Subtitle>Free Premium Extenstion</Card.Subtitle>
+                    </Card.Header>
+                    <Card.Body>
+                        While writing this hypixel conducts maintinance to its servers. Since our <Link to="/premium">premium plan</Link> isn't of much use when there are no new auctions, we are extending the subscriptions of our premium users for as long as this maintinance will last/lasted.
```

Comment:
Just copying from the blog doesn't work. It could take months until this is updated again.  
This should be replaced/extended with a link to the article in the blog.
You could show the latest post from category skyblock here. That would add extra latency.

Reply:
` While writing this hypixel conducts maintinance to its servers` makes no sense without the context of the blog. remove it.